src/util/options.schema.json

```diff
@@ -954,6 +954,19 @@
                 "Whether the node at which a thread is created is part of its threadid",
               "type": "boolean",
               "default" : true
+            },
+            "context": {
+              "title": "ana.thread.context",
+              "type": "object",
+              "properties": {
+                "create-edges": {
+                  "title": "ana.thread.context.create-edges",
+                  "description": "threadID analysis: Encountered create edges in context.",
+                  "type": "boolean",
+                  "default": true
```

Comment:
Isn't this option a bit redundant? It's also misleading because it's `true` by default, but actually that on its own doesn't do anything and also requires threadCreateEdges analysis to be activated.

Seems like the analysis activation could be enough of a toggle. 

---

**EDIT:** Given my confusion in the other comment, it now seems that maybe there should only be this option and no extra analysis?

Reply:
This is true by default, as are many context-sensitivity options. Disabling this makes the analysis less precise. If `threadCreateEdges` is enabled some of the precision can be recovered again.
In this way, it is analogous to the mechanism for improving int values for partial contexts Felix implemented for base.

Reply:
Right, so there's three levels of precision.